include/picongpu/param/fieldSolver.param

```diff
@@ -57,11 +57,18 @@ namespace fields
     /** FieldSolver
      *
      * Field Solver Selection:
-     *  - Yee< CurrentInterpolation > : standard Yee solver
-     *  - YeePML< CurrentInterpolation >: standard Yee solver using Perfectly Matched Layer Absorbing Boundary Conditions (PML)
+     *  - Yee< CurrentInterpolation > : Standard Yee solver approximating derivatives with respect to time and space
+     *                                  by second order finite differences.
+     *  - YeePML< CurrentInterpolation >: Standard Yee solver using Perfectly Matched Layer Absorbing Boundary Conditions (PML)
      *  - Lehe< CurrentInterpolation >: Num. Cherenkov free field solver in a chosen direction
      *  - LehePML< CurrentInterpolation >: Num. Cherenkov free field solver in a chosen direction
      *                                     using Perfectly Matched Layer Absorbing Boundary Conditions (PML)
+     *  - DirSplitting< CurrentInterpolation >: Sentoku's Directional Splitting Method
+     *  - ArbitraryOrderFDTD< CurrentInterpolation, 4 >: Solver using 4 neighbors to each direction for the approximation of
+     *                                                   *spatial* derivatives by finite differences.
+     *                                                   The number of neighbors can be changed from 4 to any positive, integer number.
+     *                                                   The order of the solver will be twice the number of neighbors in each direction.
+     *                                                   Yee's method is a special case of this using one neighbor to each direction.
      *  - None< CurrentInterpolation >: disable the vacuum update of E and B
      */
     using Solver = maxwellSolver::Yee< CurrentInterpolation >;
```

Comment:
I think it would be best to also enable the new solver in at least one setup for CI so that it is compile-time tested.

Reply:
Ok. How to do this? Just changing the solver in one of the examples or in a `cmakeFlags`?

Reply:
Yes. The idea is that it is part of some .param + cmakeFlags configuration in at least either an example or test. For which example it fits, I think you know better. In case none, can add to test [here](https://github.com/ComputationalRadiationPhysics/picongpu/blob/dev/share/picongpu/tests/compileFieldSolver/cmakeFlags)